main.py

```diff
@@ -35,13 +32,17 @@ def init_logging(default_level=logging.INFO):
         logging.basicConfig(level=default_level)
     logging.info('Successfully initialized logger.')
 
+#Start logging bfore creating the application so that __init__.py log will stay in logging_dev/loggin_prod.
```

Comment:
Put a space between # and first word (a rule of thumb is to follow the formatting of the rest of the project. Also typo bfore. Also please reword as `__init__.py will log to logging_dev/logging_prod.`